research/blueprint_algo/blueprint_short_deck_poker.py

```diff
@@ -1,64 +1,43 @@
 """
-I think this is right??? This is tough!
+call python blueprint_short_deck_poker.py in the research/blueprint_algo directory
 
-call python blueprint_kuhn.py in the blueprint_algo directory
-
-The following code is an attempt at mocking up the pseudo code for the blue print algorithm found in Pluribus.
+The following code is an attempt at mocking up the pseudo code for the blueprint algorithm found in Pluribus.
 -- That pseudo code can be found here, pg. 16:
 ---- https://science.sciencemag.org/content/sci/suppl/2019/07/10/science.aay2400.DC1/aay2400-Brown-SM.pdf
 
-Additionally, this code has been applied to Kuhn poker and left in a functional style that mimics code that has been
-pushed to the feature/add-kuhn-poker branch. That code was inspired by the two links below, I believe, pg 12:
+Additionally, this blueprint mockup has previously been applied to Kuhn poker.
+-- https://github.com/fedden/pluribus-poker-AI/blob/develop/research/blueprint_algo/blueprint_kuhn.py
+
+This code was left in a functional style that mimics code that has been pushed to the feature/add-kuhn-poker branch.
+That code was originally inspired by the two links below, I believe, pg 12:
```

Comment:
Really nice, thanks for all this documentation.

Reply:
Sure, so I don't lose my mind!